__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require('supertest');
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const app = require('../app');
+const { response } = require('../app');
+
+
+beforeEach( () => seed(testData));
+afterAll( ()=> db.end());
+
+
```

Comment:
Would be good to add an error handling test for an incorrect path, eg. /api/banana.

Reply:
ok